internal/utils/env/env.go

```diff
@@ -0,0 +1,37 @@
+package env
```

Comment:
nit: should we call the package `moreos` and API `LookupEnv` :) ?

Reply:
`moreos` sounds like handle the different between platform, like your guys do in [func-e](https://github.com/tetratelabs/func-e/tree/master/internal/moreos) project.

Reply:
cool, valid argument